### PR TITLE
Feat: Homepage

### PR DESCRIPTION
**Pull request by Logan1x**
This PR includes a homepage design for Ecomm App.
The Changes includes,
- Adding Images
- Defining Global CSS rules
- HTML file for homepage

Adding Screenshot below for reference

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+          <p>Groceries</p>
+        </div>
+        <div class="hero-card-category">
+          <img
+            src="https://rukminim1.flixcart.com/flap/128/128/image/29327f40e9c4d26b.png?q=100"
+            alt=""
+          />
+          <p>Groceries</p>
+        </div>
+      </div> -->
```
Comment (romabulani) [inline index.html]: You should remove the comments if not required

**Review by MurtazaBagwala07 — APPROVED** [review]
Looks good to merge , maybe add cursor:pointer to product cards

**Review by romabulani — APPROVED** [review]
Remove the comments which are not needed in HTML and CSS files
Rest all looks good to merge

**Comment by Logan1x** [comment]
Done the changes, Thanks @romabulani & @MurtazaBagwala07 :)

**Pull request merged**